fido/pronomutils.py

```diff
@@ -15,9 +15,12 @@
 # FIDO uses the UK National Archives (TNA) PRONOM File Format and Container descriptions.
 # PRONOM is available from http://www.nationalarchives.gov.uk/pronom/
 #
+from __future__ import absolute_import
 import sys
 from xml.dom import minidom
-__pronomutils__ = {'version' : '1.0.1'}
+
+from . import __version__
```

Comment:
Fido version and pronomutils version used to be different - is it intentional to make them the same now?